doc/source/rllib/index.rst

```diff
@@ -99,34 +99,74 @@ Install RLlib and `PyTorch <https://pytorch.org>`__, as shown below:
 .. note::
 
     To be able to run the Atari or MuJoCo examples, you also need to do:
-    `pip install "gymnasium[atari,accept-rom-license,mujoco]"`.
 
-This is all. You can now start coding against RLlib. Here is an example for running the PPO Algorithm on the
+    .. code-block:: bash
+
+        pip install "gymnasium[atari,accept-rom-license,mujoco]"
+
+This is all, you can now start coding against RLlib. Here is an example for running the :ref:`PPO Algorithm <ppo>` on the
 `Taxi domain <https://gymnasium.farama.org/environments/toy_text/taxi/>`__.
-You first create a `config` for the algorithm, which defines the RL environment and
-any other needed settings and parameters.
+You first create a `config` for the algorithm, which defines the :ref:`RL environment <rllib-key-concepts-environments>` and any other needed settings and parameters.
+
+.. testcode::
+
+    from ray.rllib.algorithms.ppo import PPOConfig
+    from ray.rllib.connectors.env_to_module import FlattenObservations
+
+    # Configure the algorithm.
+    config = (
+        PPOConfig()
+        .environment("Taxi-v3")
+        .env_runners(
+            num_env_runners=2,
+            # Observations are discrete (ints) -> We need to flatten (one-hot) them.
+            env_to_module_connector=lambda env: FlattenObservations(),
+        )
+        .evaluation(evaluation_num_env_runners=1)
+    )
+
+
+Next, ``build`` the algorithm and ``train`` it for a total of five iterations.
+One training iteration includes parallel, distributed sample collection by the
+:py:class:`~ray.rllib.env.env_runner.EnvRunner` actors, followed by loss calculation
+on the collected data, and a model update step.
+
+.. testcode::
+
+    from pprint import pprint
+
+    # Build the algorithm.
+    algo = config.build_algo()
+
+    # Train it for 5 iterations ...
+    for _ in range(5):
+        pprint(algo.train())
+
+At the end of your script, you evaluate the trained Algorithm:
+
+.. testcode::
+
+    # ... and evaluate it.
+    pprint(algo.evaluate())
+
 
-Next, `build` the algorithm and `train` it for a total of five iterations.
-One training iteration includes parallel, distributed sample collection by the :py:class:`~ray.rllib.env.env_runner.EnvRunner` actors,
-followed by loss calculation on the collected data, and a model update step.
+.. testcode::
+    :hide:
 
-At the end of your script, RLlib evaluates the trained Algorithm:
+    algo.stop()
```

Comment:
Haha. Yes that is needed.

Reply:
We might however show it explicitly as otherwise users might run into problems.

Reply:
... in their own code

Reply:
Great idea. Will add a one-liner for this API.

Reply:
done